lib/x/client.rb

```diff
@@ -31,11 +31,12 @@ def initialize(bearer_token: nil,
       debug_output: nil,
       array_class: ResponseHandler::DEFAULT_ARRAY_CLASS,
       object_class: ResponseHandler::DEFAULT_OBJECT_CLASS,
-      max_redirects: RedirectHandler::DEFAULT_MAX_REDIRECTS)
+      max_redirects: RedirectHandler::DEFAULT_MAX_REDIRECTS,
+      proxy_url: nil)
```

Comment:
Small suggestion: The order is technically irrelevant, but I would move this up to follow `write_timeout`, so the `Connection` options are all close together.